src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const bigNumberOne = Big(numberOne);
+  let result;
+
+  switch (operation) {
+    case '+':
+      result = bigNumberOne.plus(numberTwo);
+      break;
+    case '-':
+      result = bigNumberOne.minus(numberTwo);
+      break;
+    case 'X':
+      result = bigNumberOne.times(numberTwo);
+      break;
+    case '÷':
+      if (numberTwo === '0') {
```

Comment:
Let's check for other cases. e.g. 0.0 which is still 0 but not `0.0 == 0` is false.

Reply:
I'll check for this and add a test for it.